docker-compose.yml

```diff
@@ -17,7 +17,8 @@ services:
         - PULUMI_VERSION # install this version of pulumi
         - PHP_VERSION # 5.6, 7.0, 7.1, 7.2, 7.3, 7.4, 8.0, 8.1
         - USE_DOT_NET # install ubuntu:latest dotnet core packages
-        - GOLANG_VERSION # install this golang version. blank for none. latest is 1.18
+        - USE_KUBECTL # install kubectl:stable
```

Comment:
Add `KUBECTL_VERSION` here. If you put `latest`, then detect latest in Dockerfile.